exp-player/addon/components/exp-frame-base.js

```diff
@@ -64,19 +64,16 @@ export default Ember.Component.extend({
             this.set('id', `${kind}-${frameIndex}`);
         }
     },
-    setupParams(params, clean) {
+    setupParams(clean) {
         // Add config properties and data to be serialized as instance parameters (overriding with values explicitly passed in)
-        params = params || this.get('frameConfig');
+       var params = this.get('frameConfig');
 
         var defaultParams = {};
         Object.keys(this.get('meta.parameters').properties || {}).forEach((key) => {
             defaultParams[key] = this.get(`meta.parameters.properties.${key}.default`);
         });
 
         Object.keys(this.get('meta.data').properties || {}).forEach((key) => {
-            if (this[key] && this[key].isDescriptor) {
```

Comment:
This check prevents computed properties from getting passed into the exp-frame. As a result, the value of `responses` in expData will not get passed into the frame, and the user will not see their previous answers. A side-effect of this is that computed properties on exp-frames will need setters.